### PR TITLE
SGX Wallet Documentation

### PR DESCRIPTION
**Pull request by ebruengwall-ant**

### CODE REVIEW

**Review by christineRPM — DISMISSED** [review]
looks good

File: validators/sgx-wallet.md [hunk 1]
```diff
@@ -0,0 +1,166 @@
...
+
+Note: if you are not using Ubuntu 18.04 (something that we do not recommend), you may need to rebuild the sgx-software-enable utility before use by typing:
+
+```bash
+cd sgx-software-enable;
+make
+cd ..
+```
+
+**Instal SGX Library:**
```
Comment (cstrangedk) [inline validators/sgx-wallet.md]: typo -> Install

File: validators/sgx-wallet.md [hunk 2]
```diff
@@ -0,0 +1,166 @@
...
+```
+#### STEP 2 - Enable SGX
+
+**SGX Wallet repository includes the sgx_enable utility. To enable SGX run:**
+
+```bash
+sudo ./sgx_enable
+```
+
+Note: if you are not using Ubuntu 18.04 (something that we do not recommend), you may need to rebuild the sgx-software-enable utility before use by typing:
```
Comment (cstrangedk) [inline validators/sgx-wallet.md]: replace we

File: validators/sgx-wallet.md [hunk 3]
```diff
@@ -0,0 +1,166 @@
...
+> ***If you already executed the previous steps please move to STEP 3***
+
+Enter BIOS by pressing the BIOS key during boot. The BIOS key varies by manufacturer and could be F10, F2, F12, F1, DEL, or ESC.
+
+Usually Intel SGX is disabled by default.
+
+***To enable:***
+
+find the Intel SGX feature in BIOS Menu (it is usually under the "Advanced" or "Security" menu)
+Set SGX in BIOS as enabled (preferrably) or software-controlled.
```
Comment (cstrangedk) [inline validators/sgx-wallet.md]: typo -> preferably

**Review by cstrangedk — CHANGES_REQUESTED** [review]
small fixes needed

File: validators/sgx-wallet.md (hunk 1 above)
Comment (ebruengwall-ant) [reply validators/sgx-wallet.md]: done

File: validators/sgx-wallet.md (hunk 2 above)
Comment (ebruengwall-ant) [reply validators/sgx-wallet.md]: changed

File: validators/sgx-wallet.md (hunk 3 above)
Comment (ebruengwall-ant) [reply validators/sgx-wallet.md]: changed

**Review by cstrangedk — DISMISSED** [review]
lgtm, thank you! 🚀

**Review by cstrangedk — APPROVED** [review]

**Review by christineRPM — APPROVED** [review]
Looks good

**Pull request merged**